src/docs/guides/microsoft-entra-id-integration/README.md

```diff
@@ -50,7 +50,7 @@ The last step is to copy Application Id and Tenant Id from Azure Portal
 
 ![image](assets/get-appid-tenantid-from-portal.jpg)
 
-[Navigate to Security/Azure Active Directory](https://localhost:5001/Admin/Settings/OrchardCore.Microsoft.Authentication.AzureAD) in OrchardCore Admin to configure the AzureAD app.
+[Navigate to Security/Microsoft Entra ID](https://localhost:5001/Admin/Settings/OrchardCore.Microsoft.Authentication.MicrosoftEntraID) in OrchardCore Admin to configure the AzureAD app.
```

Comment:
This is an empty page for me:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/4ed4010a-21e3-4664-bf97-3d874f71292e)

Reply:
Let me check ..

Reply:
Still the case.

Reply:
This is resolved https://github.com/OrchardCMS/OrchardCore/pull/14015#discussion_r1270471772

Reply:
I revert the change because changing the feature name will be a breaking change. Right?

Reply:
Changing the URL here is fine, if it's just the URL, but yes, the feature's technical ID shouldn't change. And the link in the docs should be correct.